src/components/ValuesScreen.tsx

```diff
@@ -1,28 +1,127 @@
 import React from 'react';
-import { View } from 'react-native';
-import { Text } from 'react-native-paper';
+import { View, StyleSheet } from 'react-native';
+import { Text, Title, List, Button} from 'react-native-paper';
 
 import { createStackNavigator } from '@react-navigation/stack';
 import { CustomNavigationBar } from './CustomNavigationBar';
+import { FlatList } from 'react-native-gesture-handler';
 
 const ValuesStack = createStackNavigator();
 
-const ViewContent = () => {
+const RelationView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Relation</Text>
+  </View>
+); 
+
+const CareerView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Studier/karrriär</Text>
+  </View>
+); 
+
+const InterestsView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Fritid/intressen</Text>
+  </View>
+);
+
+const MindView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Sinne/Kropp/Spirituellt</Text>
+  </View>
+);
+
+const ResposibilityView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Dagliga ansvar</Text>
+  </View>
+);
+
+const SupportView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Stödpersoner</Text>
+  </View>
+);
+
+
+const CreateButton = (props: any) => {
   return (
-    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-      <Text>Values</Text>
+    <View style={{flex: 0.14, alignItems: 'center', justifyContent: 'center'}}>
+      <Button theme={{ roundness: 30 }} contentStyle={{width: 240, height: 70}} compact={true} mode="outlined" onPress={props.function}>
+        <Text>{props.name}</Text>
+      </Button>
     </View>
-  );
+      )
 }
 
+
+  
+const ViewContent = ({navigation}: any) => {
+  
+  const relationButton = () => {
+    navigation.navigate('Relation');
+  };
+
+  const careerButton = () => {
+    navigation.navigate('Career');
+  };
+
+  const interestsButton = () => {
+    navigation.navigate('Interests');
+  };
+
+  const mindButton = () => {
+    navigation.navigate('Mind');
+  };
+
+  const responsibilityButton = () => {
+    navigation.navigate('Responsibility');
+  };
+
+  const supportButton = () => {
+    navigation.navigate('Support');
+  };
+
+
+  return (
+    <View style={{flex: 1}}>
+    <View style={{flex: 0.10, alignItems: 'center', justifyContent: 'center'}}>
+      <Title>Värdering</Title>
+    </View>
+    <CreateButton name="relation" function={relationButton}/>
```

Comment:
Name should come from language component and depend on the choice from settings

Reply:
Yes we have fixed that now.

Reply:
I'm looking through this right now and I don't think you have pushed the most recent changes ("Värdering" is still used instead of the key valuesHeaderEvaluation from dictionary). If you are still working with other changes then feel free to push it up once it is ready, I'll look through it then to avoid outdated comments.

Reply:
Yes you are right, we have only fixed it on the things we are done with so far.